dartsim/BUILD.bazel

```diff
@@ -47,12 +47,11 @@ public_headers = [
 ]
 
 cc_binary(
-    name = "libignition-physics2-dartsim.so",
```

Comment:
Was there any reason this was renamed?  There are tests that seem to expect this naming that begin to fail.

Reply:
I see that some tests in `ign-gazebo` look for a `.so` named `libignition-physics-dartsim-plugin.so` but some tests within `ign-physics` look for a `.so` named `libignition-physics2-dartsim.so`, perhaps we just generate them both?

Reply:
Yes, that was the issue.

Reply:
I just revisited this, it's because I injected the `libignition-physics2` prefix in the test fixture.  I corrected this back to the non-versioned name, as Gazebo expects in 92e5427